### PR TITLE
Added mutablerecords in setup.py

### PR DESCRIPTION
**Pull request by trishnaguha**
Fixes https://github.com/google/openhtf/issues/48

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -52,6 +52,7 @@ def run(self):
     'singledispatch==3.4.0.3',
     'six==1.9.0',
     'Werkzeug==0.10.4',
+    'mutablerecords==0.2.4',
```
Comment (jettisonjoe) [inline setup.py]: I would follow the convention of specifying requirements in alphabetical order.

**Comment by jettisonjoe** [comment]
Hello, and thanks for the contribution! I'm Joe, the lead maintainer of OpenHTF. I've reviewed and commented on your PR.

If you're planning to use OpenHTF, I'd love to hear about your use case and how you found us.

**Comment by jettisonjoe** [comment]
#51 addresses the same issue. I will merge whichever of the two PR's is ready first.

**Comment by trishnaguha** [comment]
I have added the requirement in alphabtical order 
thanks @jettisonjoe :)

**Comment by jettisonjoe** [comment]
Thanks, Trishna! Your PR has been merged.

I'm still curious how you found out about the project as we haven't done anything to advertise it yet.

**Comment by trishnaguha** [comment]
Thanks @jettisonjoe  :)
I have found the project [here](https://github.com/google) and found it interesting and went for it. :)

**Pull request merged**